config/api/src/main/java/org/jboss/windup/config/metadata/LabelsetMetadata.java

```diff
@@ -0,0 +1,12 @@
+package org.jboss.windup.config.metadata;
+
+public interface LabelsetMetadata
```

Comment:
Please add comments to describe the class and the methods as well.

Reply:
done